PendingReleaseNotes

```diff
@@ -47,144 +56,156 @@
   * system
   * truncated
   * user_stats_sync
-* RGW: The beast frontend's HTTP access log line uses a new debug_rgw_access
-  configurable. This has the same defaults as debug_rgw, but can now be controlled
-  independently.
+
+* RGW: The beast front end's HTTP-access log line uses a new ``debug_rgw_access``
+  configurable. This configurable has the same defaults as ``debug_rgw``, but
+  can now be controlled independently.
+
 * RBD: The semantics of compare-and-write C++ API (`Image::compare_and_write`
-  and `Image::aio_compare_and_write` methods) now match those of C API.  Both
-  compare and write steps operate only on `len` bytes even if the respective
-  buffers are larger. The previous behavior of comparing up to the size of
-  the compare buffer was prone to subtle breakage upon straddling a stripe
-  unit boundary.
-* RBD: compare-and-write operation is no longer limited to 512-byte sectors.
-  Assuming proper alignment, it now allows operating on stripe units (4M by
+  and `Image::aio_compare_and_write` methods) now match the semantics of C
+  API. Both the compare step and the write step operate only on ``len`` bytes
+  even if their associated buffers are larger. The previous behavior of
+  comparing up to the size of the compare buffer was prone to subtle breakage
+  when straddling a stripe unit's boundary.
+
+* RBD: The compare-and-write operation is no longer limited to 512-byte sectors.
+  Given proper alignment, it is now possible to operate on stripe units (4 MB by
   default).
-* RBD: New `rbd_aio_compare_and_writev` API method to support scatter/gather
-  on both compare and write buffers.  This compliments existing `rbd_aio_readv`
-  and `rbd_aio_writev` methods.
-* The 'AT_NO_ATTR_SYNC' macro is deprecated, please use the standard 'AT_STATX_DONT_SYNC'
-  macro. The 'AT_NO_ATTR_SYNC' macro will be removed in the future.
-* Trimming of PGLog dups is now controlled by the size instead of the version.
-  This fixes the PGLog inflation issue that was happening when the on-line
-  (in OSD) trimming got jammed after a PG split operation. Also, a new off-line
-  mechanism has been added: `ceph-objectstore-tool` got `trim-pg-log-dups` op
-  that targets situations where OSD is unable to boot due to those inflated dups.
-  If that is the case, in OSD logs the "You can be hit by THE DUPS BUG" warning
-  will be visible.
+
+* RBD: New ``rbd_aio_compare_and_writev`` API method to support scatter/gather
+  on both compare and write buffers. This method complements existing
+  ``rbd_aio_readv`` and ``rbd_aio_writev`` methods.
+
+* The ``AT_NO_ATTR_SYNC`` macro is deprecated and will be removed in the future.
+  Use the standard ``AT_STATX_DONT_SYNC`` macro.
+
+* Trimming of PGLog dups is now determined by the size rather than the version.
```

Comment:
This fix has already been backported to pacific and quincy, is it still worth mentioning here @rzarzynski @neha-ojha ?